python/tank/deploy/tank_commands/desktop_migration.py

```diff
@@ -51,6 +51,11 @@ def run_interactive(self, log, args):
 
         pc = self.tk.pipeline_configuration
 
+        if pc.is_managed():
```

Comment:
cool, if you say so! Maybe i was going over this too quickly! :) I'll take it out, makes sense!